README.md

```diff
@@ -144,7 +118,11 @@ Whether to use a dark styled keyboard on iOS
 
 1. Apps are now served from HTTP on Android.
 
-    * The origin for requests from the Web View is `http://localhost:8080`.
+    * The default origin for requests from the WebView is `http://localhost:8080`. If `WKPort` preference is set, then origin will be  `http://localhost:WKPortValue`.
+
+1. Apps are now served from `ionic://` scheme on iOS.
```

Comment:
This section is a little confusing. Can you more clearly mark that the port stuff is Android only? On that note: should `WKPort` be removed and instead just use a setting like `AndroidHostname` which they could add the port if they want?